### ISSUE TITLE
[Question] matter scanner 

### ISSUE THREAD

**Issue by ChilledDayz** [simeonradivoev/MatterOverdrive#652]
Hi sorry for asking here, but I couldn't find anywhere that seems to have information. I've made a matter scanner but it doesn't scan anything, what am I doing wrong?  
Version 1.10.2 
Thanks in advance for any help

**Comment by 63502** [comment]
Now that you mention it, I would also like to know. I never actually got one to work in either the 1.10.2 or 1.7.10 versions.

**Comment by Muirrum** [comment]
Do you have the block, or the tool.

**Comment by 63502** [comment]
I think he means the tool? I know how the block works.

**Comment by ChilledDayz** [comment]
Yeah I mean the hand tool, it just shows a square over the item I'm trying to scan. It seems to start scanning water but never completes the scan.

**Comment by Muirrum** [comment]
Does this issue only occur when you are trying to scan water?

**Comment by ChilledDayz** [comment]
Yes if I try to scan any blocks it just won't do anything. But scanning water it just seems to keep looping the scan.

**Comment by Muirrum** [comment]
Well, it's not designed to replicate water, just water buckets. 
As for scanning other blocks, I'll look into that. My hands are full right now, but maybe later.
